pkg/localkube/apiserver.go

```diff
@@ -86,7 +86,10 @@ func StartAPIServer(lk LocalkubeServer) func() error {
 }
 
 func readyFunc(lk LocalkubeServer) HealthCheck {
-	hostport := net.JoinHostPort("localhost", strconv.Itoa(lk.APIServerPort))
-	addr := "https://" + path.Join(hostport, "healthz")
-	return healthCheck(addr, lk)
+	if lk.APIServerInsecurePort != 0 {
```

Comment:
see comments below

Reply:
changed to only have one invocation of healtCheck